ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -186,11 +188,20 @@ spec:
           value: "$(tasks.operator-validation.results.bundle_version)"
         - name: is_latest
           value: $(params.is_latest)
+        - name: pyxis_url
+          value: "$(tasks.set-env.results.pyxis_url)"
+        - name: pyxis_cert_path
+          value: /workspace/pyxis-ssl-credentials/operator-pipeline.pem
```

Comment:
Why are some tasks updated to use `$(workspaces.pyxis-ssl-credentials.path)` but not this one?